packages/react/src/patterns/sections/CardSection/CardSection.js

```diff
@@ -63,40 +61,44 @@ const CardSection = ({ heading, cards, theme }) => {
   };
 
   return (
-    <section
-      data-autoid={`${stablePrefix}--card-section`}
-      className={classNames(`${prefix}--card-section`, _setTheme(theme))}>
-      <div className={`${prefix}--card-section__container`}>
-        <div className={`${prefix}--card-section__row`}>
-          <h2 className={`${prefix}--card-section__heading`}>{heading}</h2>
-          <div className={`${prefix}--card-section__cards`} ref={containerRef}>
-            <div
-              className={`${prefix}--card-section__cards__row ${prefix}--row--condensed`}>
-              {cards.map((card, index) => {
-                return (
-                  <div className={`${prefix}--card-section__cards__col`}>
-                    <Card
-                      key={index}
-                      image={card.image}
-                      title={card.heading}
-                      eyebrow={card.eyebrow}
-                      copy={card.copy}
-                      href={card.cta.href}
-                      target={card.cta.target}
-                      icon={ArrowRight20}
-                      type="link"
-                    />
-                  </div>
-                );
-              })}
-            </div>
-          </div>
-        </div>
-      </div>
-    </section>
+    <ContentSection
+      heading={heading}
+      customClassName={classNames(`${prefix}--card-section`, _setTheme(theme))}
+      ref={containerRef}>
```

Comment:
Would this `ref` be better served inside the `_renderCards` method?

Reply:
okay